### PR TITLE
Prefixed table names with replicache_

### PR DESCRIPTION
**Pull request by nbarraud**
I prefixed the `space`, `entry`, `client` and `meta` tables with `replicache_` to avoid conflicts with existing userland tables.

I haven't touched the version number but I think the major should be bumped by one since that's a breaking change.

### CODE REVIEW

**Comment by nbarraud** [comment]
I haven't touched the version number but I would think the major should be bumped one since that's a breaking change.

**Comment by cesara** [comment]
thanks for this @nbarraud. I'm going to test this with the replicache-quickstarts repo first and if everything is working fine we'll merge it in.

File: src/backend/data.test.ts [hunk 1]
```diff
@@ -281,7 +281,7 @@ test("createSpace", async () => {
   ];
   for (const c of cases) {
     await withExecutor(async (executor) => {
-      await executor(`delete from space where id = 'foo'`);
+      await executor(`delete from replicache_space where id = 'foo'`);
```
Comment (cesara) [inline src/backend/data.test.ts:284]: this test is failing, I believe you need to change line 295 to 
```
        expect(String(e)).contains(
          `duplicate key value violates unique constraint "replicache_space_pkey`
        );
```

**Review by cesara — CHANGES_REQUESTED** [review]
please fix the failing test. thanks!

**Comment by cesara** [comment]
@nbarraud reopened with the fixes, Kept your commit history in the new PR https://github.com/rocicorp/replicache-express/pull/20 . Thanks again

**Pull request closed without merging**